shimmer/src/main/java/com/valentinilk/shimmer/ShimmerModifier.kt

```diff
@@ -17,23 +17,27 @@ import androidx.compose.ui.platform.debugInspectorInfo
 import kotlinx.coroutines.flow.collect
 
 fun Modifier.shimmer(
-    customShimmer: Shimmer? = null
+    customShimmer: Shimmer? = null,
+    enabled: Boolean = true
 ): Modifier = composed(
     factory = {
-        val shimmer = customShimmer ?: rememberShimmer(ShimmerBounds.View)
+        if (enabled) {
```

Comment:
I think it should be 
`fun Modifier.shimmer() : Modifier = if (enabled) composed(...) else Modifier`

Reply:
I updated the PR for use the custom shimmer approach for achieve the enabled disabled use case,
maybe this use case is common enough to have the solution in the library.